packages/hooks/src/useRemovePasskey.ts

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import { startAuthentication } from '@simplewebauthn/browser';
+import { useInvalidateQuery } from '@deriv/api';
+import { WS } from '@deriv/shared';
+
+type TError = { code?: string; name?: string; message: string };
+
+const useRemovePasskey = ({ onSuccess }: { onSuccess: () => void }) => {
+    const invalidate = useInvalidateQuery();
+
+    const [passkey_removing_error, setPasskeyRemovingError] = useState<TError | null>(null);
+
+    const removePasskey = async (id: number, passkey_id: string) => {
```

Comment:
how is id and passkey_id different?

Reply:
@amina-deriv , there are different BE data. one is just id, like number in a row. the second is unique and based on public key coming from authenticator, if I'm not mistaken